File: sample-code/src/test/java/chapter03/aaa/CalculatorTest.java
package chapter03.aaa;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class CalculatorTest {

    @Test
    public void sumOfTwoNumbers() {
        double first = 10;
        double second = 20;
        Calculator calculator = new Calculator();

        double result = calculator.sum(first, second);

        Assertions.assertEquals(30, result);
    }

}

File: src/test/kotlin/chap3/CalculatorTest.kt
package chap3

import org.junit.jupiter.api.AfterEach
import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

class CalculatorTest {

//    private lateinit var _sut: Calculator
//
//    @BeforeEach
//    fun setUp() {
//        _sut = Calculator()
//    }

    @Test
    fun sum_of_two_numbers() {
        val first: Double = 10.0
        val second: Double = 20.0
        val calculator = Calculator()

        val result = calculator.sum(first, second)

        assertEquals(30.0, result)

//        val sut = calculator.sum(first, second)

//        assertEquals(30.0, sut)
    }
}